src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -356,9 +356,10 @@ public function getGlobalScopes()
 	 * Register an observer with the Model.
 	 *
 	 * @param  object|string  $class
+	 * @param  int            $priority
```

Comment:
Oh that's odd, I just checked the syntax in other docblocks and they are aligned as well (see line 451 in Eloquent/Model.php)...